modules/core/src/main/scala/gem/config/GcalConfig.scala

```diff
@@ -22,6 +22,12 @@ object GcalConfig {
   type GcalArcs = OneAnd[ISet, GcalArc]
   type GcalLamp = GcalContinuum \/ GcalArcs
 
+  def mkLamp(continuum: GcalContinuum): GcalLamp =
+    continuum.left
+
+  def mkLamp(arc0: GcalArc, arcs: GcalArc*): GcalLamp =
+    OneAnd(arc0, ISet.fromList(arcs.toList)).right
+
```

Comment:
I think I would prefer something like this to avoid awkward naming and overloading.

```scala
object GcalLamp {

  // most general constructor
  def apply(continuum: Option[GcalContinuum], arcs: (GcalArc, Boolean)*)

  // others
  def fromContinuum(continuum: GcalContinuum): GcalLamp =...
  def fromArcs(arc0: GcalArc, arcs: GcalArc*): GcalLamp = ...

}
```

There's a point at which we'll want to newtype the `\/`.